omnibus/config/software/datadog-agent.rb

```diff
@@ -21,6 +21,8 @@
 build do
   license :project_license
 
+  bundled_agents = ["process-agent", "security-agent", "system-probe"]
```

Comment:
To avoid the multiple `if <platform>? or (not bundled_agents.include? "<binary>")` conditions in the following code, would it be simpler to start by defining `bundled_agents` for each platform? Like:
```
  bundled_agents = []
  if linux_target?
    bundled_agents = ["process-agent", "security-agent", "system-probe"]
  end
```

That way, it is also more explicit to the reader that the single binary is currently only in use on Linux.